src/ess/reduce/widgets/_optional_widget.py

```diff
@@ -64,3 +65,27 @@ def value(self, value: Any) -> None:
         else:
             self._option_box.value = self.name
             self.wrapped.value = value
+
+    def set_fields(self, new_values: dict[str, Any]) -> None:
+        new_values = dict(new_values)
+        # Set the value of the option box
+        opted_out_flag = (
+            new_values.pop(  # We assume ``opted-out`` is not used in any wrapped widget
+                'opted-out',
```

Comment:
Is `opted-out` a safe enough name? I seems rather safe, but should we make it even less likely to clash? e.g. `essreduce-opted-out` or something like that? (in the past with plopp we often added the name of the library in such attributes).

I'm guessing the `opted-out` is never seen by users, so the cleanliness of the name doesn't really matter?